go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/containers/podman/v5
 
-go 1.21
+go 1.21.0
 
 // Warning: Ensure the "go" and "toolchain" versions match exactly to prevent unwanted auto-updates
-toolchain go1.21.0
+toolchain go1.22.5
```

Comment:
That comment suggests that there's something unwanted happening here.

Reply:
Removed the toolchain and moved the comment, which we also had to do in containers/common.